src/verinfast/agent.py

```diff
@@ -856,6 +860,11 @@ def main():
         print(f"""The log for this run has moved to:
               {d}/{new_folder_name}/{new_file_name}""")
 
+    # We only do this if you have a remote config but didn't upload
+    if agent.config.shouldUpload is False and agent.config.is_path_remote():
+        print("To upload results from this location please run")
```

Comment:
Minor: Can we put a few newlines in the beginning and change it to "To upload results from this scan, please run this command:"

Reply:
I fixed this.